package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "geostyler-cql-parser",
-  "version": "4.1.0",
```

Comment:
I guess this was an accident. :) Please undo as we have an automated release workflow.